lib/modules/datasource/docker/readme.md

```diff
@@ -3,3 +3,11 @@ This datasource identifies an image's source repository according to the [pre-de
 This datasource looks for the metadata of the **latest stable** image found on the Docker registry and uses the value of the label `org.opencontainers.image.source` and `org.label-schema.vcs-url` as the `sourceUrl`.
 
 The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, therefore `org.opencontainers.image.source` label should be preferred.
+
+If you maintain a Docker image and want Renovate to find your changelogs, add a `org.opencontainers.image.source` field to your Dockerfile.
+The link must point to your GitHub or GitLab repository.
```

Comment:
From what repositories can Renovate fetch Docker image release notes?

Reply:
This looks fine to me